plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeConnectorSmokeTest.java

```diff
@@ -282,6 +282,15 @@ public void testCreatePartitionedTable()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testPathUriDecoding()
```

Comment:
I've changed the test to verify rows. I found the test to be useful when debugging this, as it can be run through the editor with breakpoints. The product tests looked quite involved to set up locally, and seem to require a Spark or Databricks cluster. Should we still remove it?

Reply:
We can set breakpoints even with product tests. Please add README.md which contains steps to create the files likes `uppercase_columns` directory if you want to add this test.